<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryForm</class>
 <widget class="QWidget" name="QueryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineQ">
      <property name="placeholderText">
       <string>query</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_q">
      <property name="text">
       <string>Proceed</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
